src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const caseTypes = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const errorMessages = {
+      errors: [
+      ],
+    };
+
+    const [originalText, queryString] = req.url.slice(1).split('?');
+
+    const params = new URLSearchParams(queryString);
+    const targetCase = params.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    if (!originalText) {
+      res.statusCode = 400;
```

Comment:
you can move all those `statusCode = 400` to https://github.com/mate-academy/node_case-transformer-server/pull/523/files#diff-aab80711ea2c23158e29927089ab5a13a1387726c2c63c17e9b19a2f6524f502R47